Externals/portaudio/CMakeLists.txt

```diff
@@ -316,8 +316,9 @@ ELSE()
 
   ENDIF()
 
-  SET(PA_PKGCONFIG_LDFLAGS "${PA_PKGCONFIG_LDFLAGS} -lm -lpthread")
-  SET(PA_LIBRARY_DEPENDENCIES ${PA_LIBRARY_DEPENDENCIES} m pthread)
+  find_package(Threads)
```

Comment:
Building Android headless client went through there and didn't like the -lpthread flag.